src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1291,6 +1297,19 @@ public static String getRegistryName(Class<? extends TileEntity> type)
     public static boolean loadAdvancements(Map<ResourceLocation, Advancement.Builder> map)
     {
         boolean errored = false;
+
+        Map<ResourceLocation, Advancement.Builder> extraAdvancements = new HashMap<>();
+        MinecraftForge.EVENT_BUS.post(new AdvancementLoadEvent(extraAdvancements));
```

Comment:
I'm not a fan of moving data back to code. Why are you doing this as an event?

Reply:
I don't want to load it via code really. I just want a way to parse the json files from somewhere and then load them with the other advancements. If you can think of a better way to do that, please let me know.